dateparser_data/settings.py

```diff
@@ -3,6 +3,7 @@
     'relative-time',
     'custom-formats',
     'absolute-time',
+    'no-spaces-time',
```

Comment:
This should probably be documented in `docs/usage.rst` (the other parsers are listed there).

Reply:
You are totally right! I added it!